ci/test-coverage.sh

```diff
@@ -1,28 +1,33 @@
 #!/usr/bin/env bash
+
 set -e
-cd "$(dirname "$0")/.."
+here=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+
+if [[ -z $CI ]]; then
+  echo "This script is used by CI environment. Use scripts/coverage.sh directly if you only want to obtain the coverage report"
+  exit 1
+fi
 
 annotate() {
   ${BUILDKITE:-false} && {
     buildkite-agent annotate "$@"
   }
 }
 
-source ci/upload-ci-artifact.sh
-source scripts/ulimit-n.sh
-
-scripts/coverage.sh "$@"
-
-if [[ -z $CI ]]; then
-  exit
-fi
+# run coverage for all
+"$here/../scripts/coverage.sh"
```

Comment:
I'd still like passsing `"$@"` for quick testing. Otherwise, we have to full run test when hacking on `ci/test-coverage.sh`...

Reply:
fixed: [2916a01](https://github.com/anza-xyz/agave/pull/1260/commits/2916a0158492fba50682ce224fad2a002155cdee)